### DISCUSSION TITLE
credentialer? - do you mean awarder?

### DISCUSSION THREAD

**Discussion by BrendaBraitling** [schemaorg/schemaorg#3320] [General]
This is an easy issue to resolve... I think... 
I noted the use of the word "credentialer"  on your website at https://schema.org/EducationalOccupationalCredential - This is not consistent with the use of the verb "award" used in the official definitions related to credentials in the body of work. 

As a technical editor and programmer... I am not sure where to express my concern that this seems to be an unnecessary and unhelpful word in the descriptions.   It may seem pedantic but is a grammatical problem related to verbs - and verbs drive the work of this entire semantic endeavor.... 

Awarders award credentials but Credentialers credential credentials?  
Peace :)

**Comment by danbri** [comment]
Thanks. Which body of work? Educational credentials specifically?

(In TV in UK Series means both “show” and “season”…. Sometimes context is
neeeded.

I am also concerned about credentials in digital signature, and Verifiable
Credentials settings. Terminology is related but rarely uniform

**Comment by danbri** [comment]
ping @BrendaBraitling 

@msporny @philbarker any thoughts?

**Comment by msporny** [comment]
Yes, lots of thoughts here -- I'll note that there is a community that meets regularly to discuss these issues wrt. education credentials, verifiable credentials, and that space in general. The VC-EDU group. /cc'ing two of that group's chairs @dmitrizagidulin @kayelle 

The terminology that the W3C Verifiable Credentials work uses for "credentialer" is "issuer"... That is, the "issuer" of the credential. That word has come about after almost a decade of difficult debate and discussion and has broad acceptance in the W3C VC work. I suggest re-using that terminology unless there is a very good reason to choose different terminology.

**Reply by BrendaBraitling** [reply]
I concur - see above... However I am only asking about what seems to be imprecise language used in the above URL.

I don't think that Schema.org intended to introduce a new entity (and perhaps a new word that the spell checker flags...) called a -- credentialer... but your prose for the entry is as follows:

"EducationalOccupationalCredential
A Schema.org Type
An educational or occupational credential. A diploma, academic degree, certification, qualification, badge, etc., that may be awarded to a person or other entity that meets the requirements defined by the **credentialer.**  "

I think the solution is to use the same construct as all the other entries.  I would propose to use  the more precise form of:

"An educational or occupational credential. A diploma, academic degree, certification, qualification, or badge awarded as a consequence of successful completion of a specified activity, course or program."

**Reply by philbarker** [reply]
Terminology here is very difficult. Given all the questions around whether the organization defining the requirements is the same as the `issuer` of the credential (in the W3C VC sense), I like @BrendaBraitling's solution. However, I would rather not emphasise academic routes to credentials (there is enough of that in the example types listed). I would prefer changing the definition to: 

> "An educational or occupational credential. A diploma, academic degree, certification, qualification, badge, etc., that may be awarded to a person or other entity that meets defined requirements."

**Reply by BrendaBraitling** [reply]
@philbarker I concur. Breaking down the silos of education/occupation is essential. I support your revision.

I apologize for the long post, but I must take a moment to air a concern regarding the difficult term ISSUER.   

I do NOT advocate for using the term ISSUER to describe an entity that awards credentials,..  Unfortunately the W3C "issuer" --  requires a VC to be transmitted - so it is NOT generalizable to all credentials. See current W3C Link below..

In my experience... 
Credentials exist beyond educational and occupational use and answer at least two questions
    1. CONTENT - What do you need to know about me? (What is my experience?)
    2. AUTHORITY - Says Who? (According to which Authority?)
 
Credentials can be awarded that are 
    1. Digital and Verifiable using W3C-VC, 
    2. Digital and not compliant with W3C-VC Verification
    4. Those that are entirely physical because the awarder does not participate in digital credentials.  
    5. ...There is also the problem of historic or pre-digital awarded credentials.  

To support the widest possible range of credentials, I advocate for decoupling  the specialized process of certifying-and-awarding a credential from the technical implementation of what happens to the credential externally and downstream.  

To be clear,  I desire all credentials to be "issued" (W3C-VC) to support sovereign control and VC, but recognize that the majority of credentials at the present time are "awarded" -- because many entities are not yet technologically capable of "issuing." 

My position is:  All credentials are awarded. Some credentials are issued. Credentials should be issued. Awarders should be Issuers,  Awarders should be able to outsource Issuing as a technical implementation.

A cursory search on "award" at Schema.org reveals it is used with frequency across the site.  I would recommend clarifying what awarding means in general and perhaps also introduce the specific case of W3C-VC issuing as desirable.  This allows progress and backward compatibility to co-exist because: Awarding is a business process, while Issuing is a technical implementation of the Awarding Process.

Thanks for the assist and your attention. 
Best Regards.
___________________
[Issuer Version 2022-0303, Current](https://www.w3.org/TR/2022/REC-vc-data-model-20220303/#dfn-issuers ) [Accessed 2023-05-23]
"issuer
A role an [entity] can perform by asserting [claims] about one or more [subjects], creating a [verifiable credential] from these [claims], 
and transmitting the [verifiable credential] to a [holder]."
